modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/DDMServiceUpgrade.java

```diff
@@ -361,7 +361,7 @@ public void register(Registry registry) {
 		registry.register(
 			"3.8.0", "3.8.1",
 			new com.liferay.dynamic.data.mapping.internal.upgrade.v3_8_1.
-				UpgradeDDMFormFieldDataSourceType(_jsonFactory));
+				UpgradeDDMFormField(_jsonFactory));
```

Comment:
@carolmariaabb, what do you think about keeping a specific name instead of a so generic one? Changing to `UpgradeDDMFormFieldDataSourceTypeAndPredefinedValue` would be an alternative?

Reply:
Good! Do you mind to rename and forward to Brian? Thanks, @carolmariaabb. :smile: